FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/startup.c

```diff
@@ -34,8 +34,9 @@
 extern void vPortSVCHandler( void );
 extern void xPortPendSVHandler( void );
 extern void xPortSysTickHandler( void );
-extern void uart_init();
-extern int main();
+extern void uart_init( void );
+extern int main( int, int );
```

Comment:
This signature of main does not match with the definition in main.c. I think you should change both to:
```c
extern int main( void );
```

Reply:
Oh, right. I've checked the other ports as well and this has been off a few times.
Good spot.